### ISSUE TITLE
NDIScopes hangs on launch

### ISSUE THREAD

**Issue by residualgravy** [MindStudioOfficial/ndiscopes#7]
V0.6.0 hangs up as it launches at the "loading settings" phase.

![image](https://user-images.githubusercontent.com/12982040/182468094-c79b343e-1d65-4960-82d6-563434428be8.png)

V0.5.1 worked great and still does.

Currently running Windows 10 Version 21H2 with an RTX 3070 Ti GPU.

I'm happy to answer any questions you might have about the issue. This is a really cool project and I'm looking forward to watching the progress.

**Comment by MindStudioOfficial** [comment]
@residualgravy 
Thanks for your Feedback :D

Could you provide the contents of the Config (`config.json`) File at `C:\Users\<Username>\AppData\Roaming\com.mindstudio\ndiscopes\`?

Is the file/folder even present?

Glad you like the project <3

**Comment by residualgravy** [comment]
The folder/file are present. The file contents:

`{"backdropOpacity":0.3,"enableWFBackdrop":false,"showWFScale":true,"vScopeScale":2.25,"wfScaleType":0,"audioLevelEnabled":true,"audioOutputEnabled":true,"scopeLayout":[0,1,2]}`

**Comment by MindStudioOfficial** [comment]
@residualgravy 
Found the Issue:

It is trying to load a setting introduced in v0.6.0 that is of course not there.
Normally I check if the setting is there and if not initialize it with a default value. 
Here I forgot that check :D

I already was able to reproduce the bug and implement a Fix for this.
The fix will be in v0.6.1

For now a **workaround** for you to be able to use v0.6.0 is to just delete the config.json file. This way it will create a new config with all the correct default values.
